user_manual/pim/sync_thunderbird.rst

```diff
@@ -10,41 +10,41 @@ Recommended method
 
-1. The `Lightning calendar <https://addons.thunderbird.net/de/thunderbird/addon/lightning/>`_ (Only needed for Thunderbird versions up to 68)
-2. `TBSync <https://addons.thunderbird.net/de/thunderbird/addon/tbsync/>`_ and
-3. The `TBSync provider for CalDAV and CardDAV <https://addons.thunderbird.net/de/thunderbird/addon/dav-4-tbsync/>`_
+1. The `Lightning calendar <https://addons.thunderbird.net/de/thunderbird/addon/lightning/>`_ (Only needed for Thunderbird versions up to 68).
+2. `TBSync <https://addons.thunderbird.net/de/thunderbird/addon/tbsync/>`_.
+3. The `TBSync provider for CalDAV and CardDAV <https://addons.thunderbird.net/de/thunderbird/addon/dav-4-tbsync/>`_.
 
-When they are installed, if you are on Windows, go to **Extras**/**Synchronisation settings (TBSync)** or **Edit/Synchronisation settings (TBSync)** if on Linux
+When they are installed, if you are on Windows, go to **Extras**/**Synchronisation settings (TBSync)** or **Edit/Synchronisation settings (TBSync)** if on Linux, and then:
 
 * In the account manager choose "**Add account** / **CalDAV / CardDAV account**"
 * In the next window, go with the default called **Automatic configuration** and click **next**
 * Enter an **account name**, which you can freely choose, **user name**, **password** and the **URL of your server** and click **next**
-* In the next window, TBSync should have autodiscovered the CalDAV and CardDAV addresses. When it has, click **Finish**.
+* In the next window, TBSync should have autodiscovered the CalDAV and CardDAV addresses. When it has, click **Finish**
 * Now check the box **Enable and synchronize this account**. TBSync will discover all address books and calenders your account has access to on the server
-* Check the box next to each calender and address book you want to have synchronised, also set how often you want them to be synchronised and push the button **sychronise now**
+* Check the box next to each calender and address book you want to have synchronised, also set how often you want them to be synchronised and push the button **sychronize now**
 * After the first successful synchronisation is complete, you can close the window. Henceforth, TBSync will do the work for you. You are done and can skip the next sections (unless you need a more advanced address book)
 
 
 Alternative: Using the CardBook add-on (Contacts only)
 ------------------------------------------------------
 `CardBook <https://addons.thunderbird.net/de/thunderbird/addon/cardbook/>`_ is an advanced alternative to Thunderbird's address book, which supports CardDAV. You can have TBSync and CardBook installed in parallel.
  
-1. Click the Cardbook icon in the upper right corner of Thunderbird:
+1. Click the CardBook icon in the upper right corner of Thunderbird:
 
 .. image:: ../images/cardbook_icon.png
 
-2. In Cardbook:
+2. In CardBook:
 
-   -  "Adressbook > New Adressbook **Remote** > Next
+   -  "Adress book > New Adress book **Remote** > Next
```

Comment:
There is a **d** missing or is there a typo in the app?

Reply:
> There is a **d** missing

Where?

See also the description of the program frame in the photo of the user manual.

Reply:
On the screenshot it is correct. But not in the text. I was talking of `address book` vs `adress book`

![image](https://user-images.githubusercontent.com/8111200/101796616-63338f80-3b09-11eb-9032-472613e59b58.png)

Reply:
Ah! My oversight 😉. I correct it immediately.